Jenkinsfile

```
@@ -756,8 +756,7 @@ pipeline {
                             filename 'Dockerfile.centos.7'
                             dir 'utils/docker'
                             label 'docker_runner'
-                            additionalBuildArgs "-t ${sanitized_JOB_NAME}-centos7 " +
-                                '$BUILDARGS_QB_TRUE' +
+                            additionalBuildArgs dockerBuildArgs(qb: quickbuild()) +
```

We shouldn't eliminate the `-t ...` argument.
```suggestion
                            additionalBuildArgs "-t  ${sanitized_JOB_NAME}-centos7 " +
                                                dockerBuildArgs(qb: quickbuild()) +
```

It's true that `-t ...` is not used consistently throughout this file.  But it is used probably a majority of the time and is a bug where it's not used.  You could add those in this PR if you wanted to.  :-)  Or not, but at least not remove it from here.

---

ok